api/src/test/scala/io/codegalaxy/api/chapter/ChapterRespDataSpec.scala

```diff
@@ -7,15 +7,16 @@ import play.api.libs.json.Json
 
 class ChapterRespDataSpec extends FlatSpec with Matchers {
```

Comment:
I guess we should rename this file/class to `ChapterWithStatisticsRespDataSpec` then as well.

Reply:
Okay